openedx/core/djangoapps/credentials/api/v1/views.py

```diff
@@ -1,3 +1,55 @@
 """
 Credentials API views (v1).
 """
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.models import User
+from django.http import HttpResponse, HttpResponseForbidden, HttpResponseBadRequest
+import json
+
+
+@login_required()
+def generate_program_credential(request):
```

Comment:
@tasawernawaz  we have a well-established convention of using Django Rest Framework to implement HTTP APIs.  Any reason not to be consistent with that for this project?